### PR TITLE
Add error popup if dependency wasn't added

### PR DESCRIPTION
**Pull request by rcd27**
Related to #4

### CODE REVIEW

File: ideplugin/src/main/kotlin/com/birbit/artifactfinder/ideplugin/BuildDependencyHandler.kt [hunk 1]
```diff
@@ -26,11 +26,19 @@ import com.intellij.openapi.actionSystem.Presentation
...
+    fun addMavenDependency(coordinate: String, onSuccess: () -> Unit, onError: () -> Unit) {
+        val parsedCoordinate = GradleCoordinate.parseCoordinateString(coordinate)
+
+        module.getModuleSystem().registerDependency(parsedCoordinate)
+
+        // There is no callback from Gradle about the fact that a dependency was registered, so we should check that
+        val resolvedDependency = module.getModuleSystem().getResolvedDependency(parsedCoordinate)
+        if (resolvedDependency != null) {
+            sync()
+            onSuccess.invoke()
```
Comment (yigit) [inline ideplugin/src/main/kotlin/com/birbit/artifactfinder/ideplugin/BuildDependencyHandler.kt]:
```suggestion
            onSuccess()
```

File: ideplugin/src/main/kotlin/com/birbit/artifactfinder/ideplugin/BuildDependencyHandler.kt [hunk 2]
```diff
@@ -26,11 +26,19 @@ import com.intellij.openapi.actionSystem.Presentation
...
+        } else {
+            onError.invoke()
```
Comment (yigit) [inline ideplugin/src/main/kotlin/com/birbit/artifactfinder/ideplugin/BuildDependencyHandler.kt]:
```suggestion
            onError()
```

**Comment by rcd27** [comment]
I've applied your suggestions, @yigit , but it still wants me to fix something :) Please, do not disappear for next two months, let's merge this PR so I can say to my friends that I've contributed to ArtifactFinder :D

**Comment by yigit** [comment]
:+1: i'll stay around no worried. I kind of get side-tracked w/ work and wasn't sure how many people are actually using the plugin, i love it though (being in the android team, we create way too many new projects to reproduce bugs etc so it help me a lot :) )

**Comment by yigit** [comment]
so i guess i'll wait to merge this? We can merge this  (if it works) and do another PR for whatever else you want to fix?

**Comment by rcd27** [comment]
It works, @yigit.

**Comment by rcd27** [comment]
I'm open to suggestions, @yigit , have no idea what is _really_ needed.

**Comment by yigit** [comment]
I was just trying this before i push a new version but it actually fails if it is added as we cannot query the model before sync.
i'll play w/ this see how we can get this data w/o sync as waiting for sync will be very unresponsive (but might be reasonable).

**Pull request merged**